Aspire Budgeting/Facilities/AppCoordinator.swift

```diff
@@ -106,6 +115,17 @@ extension AppCoordinator {
     self.selectedFile = file
     fileValidator.validate(file: file, for: self.user!)
   }
+
+  func dashboardRefreshCallback() {
+    self.contentProvider.getDashboard(for: self.user!,
+                                      from: self.selectedFile!,
+                                      using: self.dataMap!)
+    {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/d5353e560168880237460fc75a2a6ee44e50449b/Aspire Budgeting/Facilities/AppCoordinator.swift#L123"></span>Opening braces should be preceded by a single space and on the same line as the declaration.<br />
<code>opening_brace</code> <code>AppCoordinator.swift:123</code></td>
    </tr>
  </tbody>
</table>